getting_started/getting_started.py

```diff
@@ -0,0 +1,257 @@
+# -*- coding: utf-8 -*-
+# %% [markdown] {"tags": ["remove-md"]}
+# # Getting Started with Snorkel
+
+# %% {"tags": ["remove-md"]}
+import os
+
+# Make sure we're running from the spam/ directory
+if os.path.basename(os.getcwd()) == "snorkel-tutorials":
+    os.chdir("getting_started")
+
+# %% [markdown]
+# ## Programmatically Building & Managing Training Data with Snorkel
+#
+# Snorkel is a system for _programmatically_ building and managing training datasets **without needing to hand-label _any_ training data**.
+# In Snorkel, users can develop training datasets in hours or days rather than hand-labeling them over weeks or months.
+#
+# Snorkel currently exposes three key programmatic operations:
+# - **Labeling data**, for example using heuristic rules or distant supervision techniques
+# - **Transforming data**, for example rotating or stretching images to perform data augmentation
+# - **Slicing data** into different critical subsets.
+# Snorkel then automatically models, cleans, and integrates the resulting training data using novel, theoretically-grounded techniques.
+
+# %% [markdown]
+# <img src="img/snorkel_ops.png" onerror="this.onerror=null; this.src='/doks-theme/assets/images/layout/TrainingData.png';" align="center" style="display: block; margin-left: auto; margin-right: auto;">
+
+# %% [markdown]
+# In this quick walkthrough, we'll preview the high level workflow and interfaces of Snorkel using a canonical machine learning problem: classifying spam.
+# We'll use a public [YouTube comments dataset](https://www.kaggle.com/goneee/youtube-spam-classifiedcomments) from Kaggle, and see how **Snorkel can enable you to train a machine learning model without _any_ hand-labeled training data!**
+# For the more detailed version, see the [Introductory Tutorial](#).
+#
+# We'll walk through five basic steps:
+#
+# 1. **Writing Labeling Functions (LFs):** First, rather than hand-labeling any training data, we'll programatically label our _unlabeled_ dataset with LFs.
+# 2. **Modeling & Combining LFs:** Next, we'll use Snorkel's `LabelModel` to automatically learn the accuracies of our LFs and reweight and combine their outputs into a single, confidence-weighted training label per data point.
```

Comment:
learn the accuracies -> learn weights?

Reply:
Technically yes this is more correct and general, but I think "accuracies" is more intuitive for walking through at this level!